spec/support/shared/spree_builders.rb

```diff
@@ -1,5 +1,4 @@
 RSpec.shared_context 'spree_builders' do
-  # TODO: This starts already to be out of hand. Think about fixing this
```

Comment:
I think that issues are constrained to GitHub and specifically this repo. What if that changes? What happens to the knowledge of that issue?